src/lib/utils/linking.js

```diff
@@ -26,3 +27,15 @@ export const openLink = async (uri: string, target: '_blank' | '_self' = '_blank
 
   return Linking.openURL(uri)
 }
+
+export const parseLinkForNavigation = (link: string) => {
```

Comment:
doesn't `react-navigation` provides any tool to analyze / parse path  ?

Reply:
react navigation has, linkTo, but it needs configured deep linking and it works only on native